api/metadata_processor_test.go

```diff
@@ -2268,6 +2299,23 @@ func (suite *MetadataProcessorSuite) assertContentUnit(metadata CITMetadata, ori
 		suite.False(missing, "Missing tag %s", x)
 	}
 
+	// likutim
+	likutim, err := models.ContentUnits(suite.tx,
```

Comment:
here I get empty array and I dont see any unit on DB.
I was try add Units type LIKUTIM [here](https://github.com/Bnei-Baruch/mdb/blob/a07d223359b6b61a9f4d8f3e8703b8b23e5f3250/migrations/data/dev.sql) but its broke other tests